proxies/live/apiproxy/resources/jsc/SetErrorVars.js

```diff
@@ -0,0 +1,210 @@
+// Regex to validate the id is a valid UUID
+const uuidRegex = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi
+const nhsNumberRegex = /^[0-9]{10}$/gi
+const dobRegex = /^[0-9]{4}-[0-9]{2}-[0-9]{2}$/gi
+
+var data = context.getVariable("request.content.data")
+// var requiredParameters = [
+//   "type",
+//   "attributes/routingPlanId",
+//   "attributes/batchReference",
+//   "attributes/recipients",
+//   "attributes/recipients/0/recipientReference",
+//   "attributes/recipients/0/nhsNumber",
+//   "attributes/recipients/0/dateOfBirth",
+//   "attributes/recipients/0/personalisation",
+// ]
+// var requestParameters = context.getVariable("")
+var seenRecipients = []
+var errors = []
```

Comment:
Its possible that we may output a response body that is actually larger than the request that came in, if there is an error with every entry in the submission.

I believe we should limit this to return the first 100 errors.